Cargo.toml

```diff
@@ -93,10 +93,12 @@ ariadne            = { version = "=0.5.0", default-features = false }
 aws-types          = { version = "=1.3.3", default-features = false }
 axum               = { version = "0.7.5" }
 axum-core          = { version = "0.4.3" }
+base64             = { version = "=0.22.1" }
 bumpalo            = { version = "=3.16.0", default-features = false }
 bytes              = { version = "1.6.0" }
 clap_builder       = { version = "=4.5.21", default-features = false, features = ["std"] }
 criterion          = { version = "=0.5.1" }
+curl               = { version = "=0.4.47" }
```

Comment:
Ok have pushed out some changes that do this, but unable to solve the `tokio` linter errors that occur from moves during the `await` function.

Reply:
Basically, the lints are saying that you cannot send `PdfDocument` across threads. This is because PDFium is _**not threadsafe**_. We basically have two options:
- Don't pass PDFDocument around in `async` functions but instead read the PDF in the function itself (see their [example about thread safety](https://github.com/ajrcarey/pdfium-render/blob/master/examples/thread_safe.rs))
- Don't care about multithreading here.

Basically this means we cannot send a document across threads in any world (which also would happen if you want to offload a task to a thread). However, this can be worked around by not sending the document but the API as a whole instead ([`Pdfium`](https://docs.rs/pdfium-render/0.8.26/pdfium_render/prelude/struct.Pdfium.html)). This would allow to use the library in a multi-threaded environment, however, the `PdfDocument` needs to stay in a single thread.

Reply:
Generally I'd say we try not passing `PdfDocument` if not needed and ignore the lint `#[expect(<lint name>, reason = "...")]` if we cannot or do not want to avoid it.